.env.sample

```
@@ -0,0 +1,7 @@
+SECRET_KEY=''3izb^ryglj(bvrjb2_y1fZvcnbky#358_l6-nn#i8fkug4mmz!''
+ALLOWED_HOSTS='*'
+DEBUG='True'
+DB_NAME='HELLO_DJANGO'
+DB_USER='U_HELLO'
+DB_PASSWORD='hA8(scA@!fg3*sc&xaGh&6%-l<._&xCf'
+DB_HOST='127.0.0.1'
```

```suggestion
DB_NAME='HELLO_DJANGO'
DB_USER='U_HELLO'
DB_PASSWORD='hA8(scA@!fg3*sc&xaGh&6%-l<._&xCf'
DB_HOST='127.0.0.1'
```
These options are not yet setup in the setting.py file. Later in the development we can give the option to use 
1. sqllite
1. any database with option to specify the database. 

So for now please remove these.